board/app/controllers/thread_controller.php

```diff
@@ -0,0 +1,111 @@
+<?php
+class ThreadController extends AppController {
+  
+  /*Create a new comment (Thread)
+  * :: - STATIC FUNCTION, can be called from the class name
+  * -> - INSTANCE, can only be called from an instance of the class.
+  */
+  public function create(){
+    $thread = new Thread();
+    $comment = new Comment();
+    $page = Param::get('page_next', 'create');
+      
+      switch ($page) 
+      {
+        case 'create':
+        break;
+          
+        case 'create_end':
+          $thread->title = Param::get('title');
+          $comment->username = Param::get('username');
+          $comment->body = Param::get('body');
+ 
+          try 
+          {
+            $thread->create($comment);
+          } 
+
+          catch (ValidationException $e) 
+          {
+            $page = 'create';
+          }
+          
+        break;
+        default:
+          throw new NotFoundException("{$page} is not found");
+        break;
+      }  
+
+    $this->set(get_defined_vars());
+    $this->render($page);
+  }
+ 
+
+  //Instance of the Thread model, and its static functions getAll()
+  public function index() 
+  {
+    $per_page = 7;
```

Comment:
I think you can set this to a const variable instead if this is represting the total number of threads to be shown per page :bow:  Just rename the variable to make it more easy to understand :dart: